package.json

```diff
@@ -61,7 +61,7 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "@types/chrome": "^0.0.231",
-    "@types/jest": "^29.5.0",
+    "@types/jest": "^29.5.1",
```

Comment:
chore (blocking): why no lock changes related to a package change? Can we run yarn and double check this is correct.

Reply:
welp, it hated my yarn.lock! Moaning about strip-ansi in node modules. Do you guys use that? It is one of the changes in the yarn.lock. It builds locally absolutely fine for me even after nuking node modules and running from fresh.